Tasks/AzureCLIV2/task.loc.json

```diff
@@ -43,7 +43,7 @@
       "required": true,
       "helpMarkDown": "ms-resource:loc.input.help.connectedServiceNameARM",
       "properties": {
-        "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+        "EndpointFilterRule": ""
```

Comment:
Why keep it empty? Can we not just remove the rule/properties section completely?

Reply:
If we're removing, it'll fallback to default value which filters out Mgmt group